src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/BasePageObject.java

```diff
@@ -233,10 +234,16 @@ public boolean verifyTitle(String title) {
     return true;
   }
 
-  public void verifyURLcontains(String givenString) {
+  public boolean isStringInURL(String exptectedString) {
     String currentURL = driver.getCurrentUrl();
-    Assertion.assertStringContains(currentURL.toLowerCase(), givenString.toLowerCase());
-    PageObjectLogging.log("verifyURLcontains", "current url is the same as expetced url", true);
+    if (currentURL.toLowerCase().contains(exptectedString.toLowerCase())) {
```

Comment:
`currentURL` contains operation result, safer option would be depending on variable sent to method.
`exptectedString.toLowerCase().contains(currentURL.toLowerCase())`